test/exercism/user_progression_test.rb

```diff
@@ -16,7 +16,9 @@ def test_user_progress_for_100_percent
       expected = {'Animal'=>[1, 1]}
       UserExercise.create(user: @user, language: 'Animal')
       actual = UserProgression.user_progress(@user)
-      assert_equal actual, expected
+      assert_equal actual.first.language, 'Animal'
+      assert_equal actual.first.user_exercises.count, 1
+      assert_equal actual.first.language_track.count, 1
```

Comment:
I think this should be 
```ruby
assert_equal expected, actual
```

Ref: http://ruby-doc.org/stdlib-1.9.3/libdoc/minitest/unit/rdoc/MiniTest/Assertions.html#method-i-assert_equal